docs/getting-started/installing-service-binding.md

```diff
@@ -8,7 +8,17 @@ sidebar_position: 1
 
 ## Operator
 
-_How can To be provided soon_
+- Navigate to the `Operators` -> `OperatorHub` in the OpenShift console
+- Select `Developer Tools` category 
+- Select the `Service Binding Operator`
+
+![Service Binding Operator](https://github.com/redhat-developer/service-binding-operator/blob/master/assets/operator-hub-sbo-screenshot.png)
+
+- Select `beta` channel
+- Select `All namespaces on the cluster` Installation mode
+- Select `Automatic` approval strategy
+- Click on `Install`
+
 
 ## Helm
```

Comment:
I don't know about Helm installation.
There is a JIRA ticket for it in our backlog - https://issues.redhat.com/browse/APPSVC-746

Reply:
Let's remove this section for now

Reply:
Done